include/range/v3/algorithm/inplace_merge.hpp

```diff
@@ -63,9 +63,9 @@ namespace ranges
                     if(len1 <= len2)
                     {
                         auto p = ranges::move(begin, middle, tmpbuf.begin()).second;
-                        merge(make_move_iterator(buf), make_move_sentinel(p.base().base()),
+                        merge(make_move_iterator(buf), make_move_iterator(p.base().base()),
```

Comment:
Unrelated drive-by changes: these arguments to `make_move_sentinel` are statically known to have the same type as the arguments passed to the corresponding `make_move_iterator`.